css/sell_media.css

```diff
@@ -316,8 +316,12 @@ a:focus {
   margin-right: 10px;
 }
 
+
 /*= Buttons
 -------------------------------------------------------------- */
+.sell-media-dialog-box .sell-media-dialog-box-content a.sell-media-button {
```

Comment:
Please make changes to sell_media.scss not sell_media.css. We compile sass to css.

Reply:
Sure, I have moved code from css to scss.